BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/NotificationServiceFake.cs

```diff
@@ -13,6 +13,17 @@ public Task SendNewUserAccountNotificationAsync(Guid hearingId, ParticipantDto p
         {
             return Task.FromResult(HttpStatusCode.OK);
         }
+
+        public Task SendNewSingleDayHearingConfirmationNotification(HearingDto hearing, IEnumerable<ParticipantDto> participants)
+        {
+            NotificationRequests = new List<AddNotificationRequest>();
```

Comment:
see the Sonar Cloud code smell - as the implementation is the same as SendNewHearingNotification, you could change this method to call that method, ie

`return SendNewHearingNotification(hearing, participants);`

Reply:
done